composer.json

```diff
@@ -1,22 +1,26 @@
 {
     "name": "pulse00/ffmpeg-bundle",
+    "type": "symfony-bundle",
     "description": "Symfony bundle to provide PHP-FFmpeg as a Symfony service (https://github.com/alchemy-fr/PHP-FFmpeg)",
-    "keywords": ["ffmpeg", "multimedia"],
+    "keywords": ["multimedia", "video processing", "video", "audio processing", "audio", "avconv", "ffmpeg", "avprobe", "ffprobe"],
     "license": "MIT",
-    "type": "symfony-bundle",
-    "require": {
-        "php-ffmpeg/php-ffmpeg": "0.5.*@dev"
-    },
-    "minimum-stability": "dev",
     "authors": [
         {
             "name": "Robert Gruendler",
             "email": "r.gruendler@gmail.com"
         }
     ],
-    "autoload": {
-        "psr-0": { "Dubture\\FFmpegBundle": "" }
+    "require": {
+        "php": "^5.3.9 || ^7.0",
+        "php-ffmpeg/php-ffmpeg": "^0.5 || ^0.6",
```

Comment:
It's safe to support PHP-FFMpeg in 0.5.x and 0.6.x versions, because between those two is no BC break. According to semver and composer semver operator `^` a 0.x release may contain BC breaks so `^0.6` is equivalent to `~0.6.0` or `>=0.6.0 <0.7.0-dev`.

This means if there is a new PHP-FFMpeg release you have the following possibilities:

* 0.7 has no BC break, thus modify the constraint to `^0.5 || ^0.6 || ^0.7` and create a 0.6.x release.
* 0.7 has a BC break, but you can support it with supporting the old versions too, modify the constraint to `^0.5 || ^0.6 || ^0.7` and create a  new 0.x.0 release.
* 0.7 has a BC break, but you can't support the old versions too, modify the constraint to `^0.7` and create a  new 0.x.0 release.